core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -477,7 +477,9 @@ class SpendReportingService(
     aggregations: Set[SpendReportingAggregationKeyWithSub]
   ): Future[SpendReportingResults] = {
     validateReportParameters(start, end)
-    requireProjectAction(project, SamBillingProjectActions.readSpendReport) {
+    requireProjectAction(project,
+                         SamBillingProjectActions.createWorkspace
```

Comment:
`readSpendReport` sounds like the right action to be checking. This probably should be a sam config change to grant `readSpendReport` to the right workspace role.

Reply:
Fixed: https://github.com/broadinstitute/rawls/pull/3193/commits/bd3cd9f1f92da9525e4e262a0778a4b893fb73a2